docs/user/ppl/functions/datetime.rst

```diff
@@ -1298,6 +1508,61 @@ Example::
     +----------------------------+
 
 
+TIME_FORMAT
+-----------
+
+Description
+>>>>>>>>>>>
+
+Usage: time_format(time, format) formats the time argument using the specifiers in the format argument.
+This supports a subset of the time format specifiers available for the `date_format`_ function.
+Using date format specifiers supported by `date_format`_ will return 0 or null.
+Acceptable format specifiers are listed in the table below.
+If an argument of type DATE is passed in, it is treated as a DATETIME at midnight (i.e., 00:00:00).
+
+.. list-table:: The following table describes the available specifier arguments.
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Specifier
+     - Description
+   * - %f
+     - Microseconds (000000..999999)
+   * - %H
+     - Hour (00..23)
+   * - %h
+     - Hour (01..12)
+   * - %I
+     - Hour (01..12)
```

Comment:
Is there a difference between `%I` and `%h`? If so, perhaps mention the difference.

Reply:
`%I` and `%h` return the same thing, they are just synonyms for each other.

Reply:
Cool! Thanks.